elastic/kibana
# [canvas] disable canvas application when there are no workpads

Closes https://github.com/elastic/kibana/issues/197370

### Test instructions
1) open new kibana installation
2) verify canvas is not available in menu or application search bar
3) use saved object import to import canvas workpad. Reload browser
4) verify canvas is available in menu and application search bar
